tools/st2ctl

```diff
@@ -231,6 +231,13 @@ function getpids(){
       done
     else
       echo "${COM} is not running"
+      if [ "$COM" == "st2sensorcontainer" ] || [ "$COM" == "sensor_container" ]
+      then
+        echo "Sensor container did not detect valid sensors to run."
+        echo "Please install sensors or check sensors configuration for existing ones."
+        echo "See http://docs.stackstorm.com/sensors.html."
+        echo "Use 'st2ctl restart-component st2sensorcontainer' to restart after you fix the problems."
```

Comment:
I would change that to be more informational: most people don't have active sensors out of box, so we need to tell them that it's an expected behavior, not necessarily a problem. 

My take on wording: 
"It is expected when no sensors are active. Learn more at http://docs.stackstorm.com/sensors.html"

@jfryman may have a better english way of saying it, let's ask him.